### DISCUSSION TITLE
Added an '!' prefix to hidden, but it does not take effect

### DISCUSSION THREAD

**Discussion by 978935221** [tailwindlabs/tailwindcss#12472] [Help]
What version of Tailwind CSS are you using?

v3.3.5

What build tool (or framework if it abstracts the build tool) are you using?

postcss 8.4.31

What version of Node.js are you using?

v20.9.0

What browser are you using?

N/A

What operating system are you using?

Windows

Reproduction URL

N/A

Describe your issue

Here's my code
```html
<div class="!hidden">
</div>
```
But it doesn't work, this class is not shown in the style above devtool.

Do not support this writing for the time being?

**Comment by wongjn** [comment]
Seems to work [in Tailwind Play](https://play.tailwindcss.com/YGG23wb1P8). This suggests there could be something incorrect with your configuration.

Consider checking:
- Tailwind has compiled since you added the class.
- The `content` file globs sufficiently cover the file that the code appears in.
- [String concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). 

Otherwise, consider providing a remotely-hosted git repository that reproduces the unexpected behavior.

**Reply by 978935221** [reply]
You see, when I put it on the website, it's not valid.

I still need to create a git repository, please wait thanks

![image](https://github.com/tailwindlabs/tailwindcss/assets/67942971/00b63c29-75d9-4ca7-8ce3-cd1d84b4f911)

**Reply by 978935221** [reply]
This is the official website of taildwindcss

It must have used taildwindcss

But if I add hidden, it will take effect and it will be hidden
